test/Xml2JsonTest.php

```diff
@@ -454,33 +455,34 @@ public function testXmlToJsonWithXmlContainingSelfClosingElement()
 
     /**
      * @group Laminas-11385
-     * @expectedException Laminas\Xml2Json\Exception\RecursionException
      * @dataProvider providerNestingDepthIsHandledProperly
+     * @param $xmlStringContents
```

Comment:
An `@param` is kinda useless if no information comes with it: can we specify the type here? perhaps adding `string` to the native type, and dropping `@param` overall?